packages/commonwealth/client/scripts/controllers/app/login.ts

```diff
@@ -208,6 +208,7 @@ export function updateActiveUser(data) {
     app.user.setEmailInterval(data.emailInterval);
     app.user.setEmailVerified(data.emailVerified);
     app.user.setJWT(data.jwt);
+    localStorage.setItem('jwt', data.jwt);
```

Comment:
This would be in either
A. browser storage + state
B. only state

to keep things simple I used localstorage here